src/app/content/utils/seoUtils.constants.ts

```diff
@@ -0,0 +1,107 @@
+// tslint:disable: object-literal-sort-keys max-line-length
```

Comment:
should name this file something to indicate that it is used in specs maybe `setUtils.spec.data.ts` or something like that